sopel/coretasks.py

```diff
@@ -137,6 +182,16 @@ def _execute_perform(bot):
 @plugin.priority('high')
 @plugin.unblockable
 def on_nickname_in_use(bot, trigger):
+    """Change the bot's nick when the current one is already in use.
+
+    This can be triggered when the bot disconnect then reconnect before the
+    server can notice a client timeout. Other reasons include mischief,
+    trolling, and obviously, PEBKAC.
+
+    This will change the current nick by adding a trailing ``_``. If a the bot
+    notices that a user with the configured nick disconnect (see ``QUIT`` event
+    handling), it will try to change it back.
```

Comment:
```suggestion
    This will change the current nick by adding a trailing ``_``. If the bot
    sees that a user with its configured nick disconnects (see ``QUIT`` event
    handling), the bot will try to regain it.
```

Reply:
"if a the bot" why did I do that here?